packages/opds-web-client/src/OPDSDataAdapter.ts

```diff
@@ -280,6 +281,24 @@ export function feedToCollection(feed: OPDSFeed, feedUrl: string): CollectionDat
     if (shelfLink) {
       shelfUrl = shelfLink.href;
     }
+
+    let aboutLabels = {
+      "about": "About",
+      "terms-of-service": "Terms of Service",
+      "privacy-policy": "Privacy Policy",
+      "copyright": "Copyright"
+    };
```

Comment:
I'm not sure where these should go.

I could've done all this in opds-feed-parser, but it's not really part of OPDS.